dll/win32/shell32/wine/control.c

```diff
@@ -736,6 +737,107 @@ static	void	Control_DoWindow(CPanel* panel, HWND hWnd, HINSTANCE hInst)
 }
 #endif
 
+#ifdef __REACTOS__
+
+/** Structure for in and out data when
+ *  search for the cpl dialog of first instance
+ */
+typedef struct tagAppDlgFindData
+{
+    PCWSTR  szAppFile;  /**< Full path to applet library as search parameter */
+    signed  sAppletNo;  /**< Number of applet in a system control library as search parameter */
+    ATOM    aCPLName;   /**< to read window property 'CPLName' */
+    ATOM    aCPLFlags;  /**< to read window property 'CPLFlags'*/
+    HWND    hRunDLL;    /**< to skip self instance */
+    HWND    hDlgResult; /**< Returned dialog handle or NULL if not found */
+} AppDlgFindData;
+
+/*
+ * Case insensitive variant of wcsstr
+ */
+static
+wchar_t * wcsistr(const wchar_t *s, const wchar_t *b)
```

Comment:
Since this is shell code, you could use StrStrIW: https://docs.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-strstriw